Makefile

```diff
@@ -63,6 +63,20 @@ CRATES := \
 	$(wildcard */*/*/*/Cargo.toml) \
 	$(wildcard */*/*/*/*/Cargo.toml)
 
+ALLMAKEFILE := \
```

Comment:
Won't this go into the 3rdparty directory too which might get weird?

Reply:
yes that's bad.